packages-native/bottom-sheet/src/components/NativeBottomSheet.tsx

```diff
@@ -50,12 +51,25 @@ export const NativeBottomSheet = (props: NativeBottomSheetProps): ReactElement =
                 {item.caption}
             </Text>
         ));
+
+        const buttonContainerStyle = { ...props.styles.modalItems?.container } as ModalItemContainerStyle | undefined;
```

Comment:
definitely cannot be `undefined`

Reply:
![image](https://user-images.githubusercontent.com/7356299/87943940-06263700-ca9f-11ea-85df-9b5b636410c9.png)

Reply:
Done